fluent-plugin-enhance-k8s-metadata/lib/fluent/plugin/filter_enhance_k8s_metadata.rb

```diff
@@ -6,7 +6,91 @@ module Plugin
     class EnhanceK8sMetadataFilter < Fluent::Plugin::Filter
       Fluent::Plugin.register_filter('enhance_k8s_metadata', self)
 
+      require_relative '../../sumologic/kubernetes/cache_strategy.rb'
+
+      helpers :record_accessor
+      include SumoLogic::Kubernetes::Connector
+      include SumoLogic::Kubernetes::Reader
+      include SumoLogic::Kubernetes::CacheStrategy
+
+      # parameters for read/write record
+      config_param :in_namespace_path, :string, default: '$.namespace'
+      config_param :in_pod_path, :string, default: '$.pod'
+      config_param :out_root, :string, default: 'kubernetes'
```

Comment:
Is this part even configurable?

Reply:
we probably need it's configurable for reuse it on metrics/log. I don't think it's changeable if we want application work